_plugins/product-data-enricher.rb

```diff
@@ -92,6 +92,7 @@ def set_cycle_label(page, cycle)
           label = template.gsub('__RELEASE_CYCLE__', cycle['releaseCycle'] || '')
           label.gsub!('__CODENAME__', cycle['codename'] || '')
           label.gsub!('__LATEST__', cycle['latest'] || '')
+          label.gsub!('__BUILDID__', cycle['buildID'] || '')
```

Comment:
hmm something like this might work : 
```ruby
          label.gsub!('__BUILDID__', "(build "+cycle['buildID']+")" || '')
```

Reply:
@jlaundry could you make pr like this and also make related changes on windowsserver and windows 's releaseLabel ?
And sorry if  I am asking too much again and again

Reply:
We can also use liquid to check if buildid is defined:

```liquid
releaseLabel: "Windows __RELEASE_CYCLE__ {% if "__BUILDID__" %}__BUILDID__{%endif%}"
```

Reply:
to be honest i was planing to make similar change to all the other products which was using that liquid checks  :)

Reply:
the main points of my suggestion  are : 
* to make things easier for  our new contributors to prepare new products ( not all of them knows programming )
* we dont need so much costumization on buidID so it will be much better to fixed as ( build NUMBER )
* making extra checks ( if statments ) might cost for our servers 2 nanosecond more work :D